.gitignore

```diff
@@ -82,3 +82,4 @@ modules.rst
 pyanaconda.*rst
 .coverage
 docs/tests/*.rst
+zanata.xml
```

Comment:
I think this also needs to be added to ``MAINTAINERCLEANFILES`` in the ``Makefile.am`` (just with the ``.in`` suffix).